static/locales/en-US.yaml

```diff
@@ -13,27 +13,27 @@ Cut: Cut
 Copy: Copy
 Paste: Paste
 Delete: Delete
-Select all: Select all
+Select All: Select All
 Toggle Developer Tools: Toggle Developer Tools
-Actual size: Actual size
-Zoom in: Zoom in
-Zoom out: Zoom out
-Toggle fullscreen: Toggle fullscreen
+Actual Size: Actual Size
+Zoom In: Zoom In
+Zoom Out: Zoom Out
+Toggle Full Screen: Toggle Full Screen
 Window: Window
 Minimize: Minimize
 Close: Close
 Back: Back
 Forward: Forward
 Right-click or hold to see history: Right-click or hold to see history
-Open New Window: Open New Window
+Open new window: Open new window
 Go to page: Go to {page}
-Close Banner: Close Banner
+Close banner: Close banner
 
-Version {versionNumber} is now available!  Click for more details: Version {versionNumber} is now available!  Click
-  for more details
-Download From Site: Download From Site
-A new blog is now available, {blogTitle}. Click to view more: A new blog is now available, {blogTitle}.
-  Click to view more
+Version {versionNumber} is now available!  Click for more details.: Version {versionNumber} is now available!  Click
+  for more details.
+Download from site: Download from site
+A new blog is now available, {blogTitle}. Click to view more.: A new blog is now available, {blogTitle}.
+  Click to view more.
```

Comment:
I would prefer to keep periods out keys if possible, as it makes using the translation strings more annoying `t('parent.key without a period')` vs `t('parent["key with . a period"]')`, I don't mind adding them to the values/the actual strings.

Off-topic: I would prefer to use shorter keys in general, even if that means that the key doesn't match the text.